### DISCUSSION TITLE
Trying to change the file location for ripped dvd files

### DISCUSSION THREAD

**Discussion by shahrulsong** [automatic-ripping-machine/automatic-ripping-machine#955] [Q&A]
I'm having an issue in changing the storage location for my dvd files. I know in this link, https://github.com/automatic-ripping-machine/automatic-ripping-machine/wiki/Docker-Troubleshooting it says not to change the arm.yaml config. But somehow, the directory is /home/arm which is non existent in my unraid server. So I tried to change it to /mnt/user/media which didn't work (not to my surprise). 

My problem is that the media is saving to a non existent file path for me which is unaccessible via the command line. What I think is going on is that it's saving in the systems folder which is my non array pool of appdata storage (nvme drive) or my server. I dont want ripped dvds to clog my mission critical nvme drive which is purely for functionality of my unraid server.

<img width="1447" alt="image" src="https://github.com/automatic-ripping-machine/automatic-ripping-machine/assets/62638731/6ac730c7-7dc6-4cf3-90f0-45dddf0b01cb">

**Comment by shitwolfymakes** [comment]
Please paste the run command or the contents of `start_arm_container.sh`, whichever you used

**Reply by shahrulsong** [reply]
<img width="1680" alt="image" src="https://github.com/automatic-ripping-machine/automatic-ripping-machine/assets/62638731/72bd7a73-1e93-48f5-8d72-5e26246ee7e1">

i dont understand why it's creating /home/arm/media/completed etc since the home directory is in the docker container itself and not saving the file into my array. This is what happens if i dont change any settings

<img width="422" alt="Snipaste_2023-10-08_10-32-32" src="https://github.com/automatic-ripping-machine/automatic-ripping-machine/assets/62638731/71ad317a-c2c1-4db2-8e6e-55b0ef31c735">

i dont save my system file into my array since its server related files

**Reply by shitwolfymakes** [reply]
Please paste the run command or the contents of `start_arm_container.sh`, whichever you used
